examples/basic_example.py

```diff
@@ -32,56 +35,79 @@ def future_loading(t, x = None):
         else:
             i = 3
         return {'i': i}
-    batt = BatteryCircuit()
+    batt = Battery()
+    initial_state = batt.parameters['x0']
+
+    # Step 2: Demonstrating state estimator
+    print("\nPerforming State Estimation Step")
+
+    # Step 2a: Setup
     ## State Estimation - perform a single ukf state estimate step
```

Comment:
Should we get rid of the "ukf" in the comment that starts with ##State Estimation - , since it could be UKF or a particle filter? (or anything else we may add in the future)